openresty-build-tools/kong-ngx-build

```diff
@@ -140,6 +144,13 @@ main() {
     fatal "OpenSSL version can not be empty"
   fi
 
+  if [[ -n "$OPENSSL_FIPS_VER" ]] && \
+       ! version_eq $OPENSSL_VER 1.0.1 && \
+       ! version_eq $OPENSSL_VER 1.0.2 ];
+  then
+    fatal "OpenSSL FIPS mode is only supported for OpenSSL 1.0.1 and 1.0.2"
```

Comment:
should we fatal or just unset the varialbe?

Reply:
Certainly not unset the variable and ignore it.